src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellItemRenderer.cs

```diff
@@ -85,7 +86,21 @@ public override AView OnCreateView(LayoutInflater inflater, ViewGroup container,
 
 			HookEvents(ShellItem);
 			SetupMenu();
-
+			var appearance = ShellContext.Shell.GetAppearanceForPivot(ShellItem);
+			if (_bottomView.Background is ColorDrawable background)
+			{
+				if (appearance is IShellAppearanceElement appearanceElement)
+				{
+					if (appearanceElement.EffectiveTabBarBackgroundColor is not null)
+					{
+						background.Color = appearanceElement.EffectiveTabBarBackgroundColor.ToPlatform();
+					}
+					else
+					{
+						background.Color = ShellRenderer.DefaultBottomNavigationViewBackgroundColor.ToPlatform();
+					}
+				}
+			}
```

Comment:
Is there a reason this is just in a `is ColorDrawable`? I there ever a chance that the drawable is something like a gradient or another type of drawable?

If this logic setting the background, should we just be set directly?

```suggestion
			if (ShellContext.Shell.GetAppearanceForPivot(ShellItem) is IShellAppearanceElement appearanceElement)
			{
				var color = appearanceElement.EffectiveTabBarBackgroundColor ?? ShellRenderer.DefaultBottomNavigationViewBackgroundColor;
				background.Color = new ColorDrawable(color.ToPlatform());
			}
```

Reply:
@mattleibow The reason I initially checked for a ColorDrawable is that the BottomNavigationView's background is of type Drawable, and it is created with a ColorDrawable (typically white) by default. There is no possibility of any other type of drawable being used by default. Therefore, we need ColorDrawable in order to change its color.